flamingoservice/stroke.go

```diff
@@ -0,0 +1,365 @@
+			return
+		}
+		if strokeClient.AuthClient.Authorize(message.GuildID, message.Author.ID, strokeCommand, "") {
+			for _, v := range message.Mentions {
+				strokes, err := strokeClient.StrokeUser(message.GuildID, message.ChannelID, v.ID)
+				ParseServiceResponse(session, message.ChannelID, strokes, err)
+			}
+		} else {
+			ParseServiceResponse(session, message.ChannelID, "<@"+message.Author.ID+"> is unauthorized to issue that command!", nil)
+		}
+	}
+}
+
+// StrokeUser adds 1 to the stroke count of a user
+func (strokeClient *StrokeClient) StrokeUser(guildID, channelID, userID string) (string, error) {
+	result, err := strokeClient.DynamoClient.UpdateItem(&dynamodb.UpdateItemInput{
+		TableName:                 aws.String(assets.StrikeTableName),
+		Key:                       buildStrokeKey(guildID, userID),
+		UpdateExpression:          aws.String("ADD strokes :s"),
+		ExpressionAttributeValues: buildStrokeUpdateExpression(1),
+		ReturnValues:              aws.String("UPDATED_NEW"),
+	})
+	if err != nil {
+		strokeClient.StrokeErrorLogger.Println(err)
+		return "", nil
+	}
+	strokeCount, ok := result.Attributes["strokes"]
+	if ok {
+		return "<@" + userID + "> has " + *strokeCount.N + " strokes.", nil
+	}
+	strokeClient.StrokeErrorLogger.Printf("stroke attribute not found after update guildID=%s userID=%s", guildID, userID)
+	return "", errors.New("stroke attribute not found after update")
+}
+
+// SuperStrokeUser adds 10 to the stroke count of a user
+func (strokeClient *StrokeClient) SuperStrokeUser(guildID, channelID, userID string) (string, error) {
+	result, err := strokeClient.DynamoClient.UpdateItem(&dynamodb.UpdateItemInput{
+		TableName:                 aws.String(assets.StrikeTableName),
+		Key:                       buildStrokeKey(guildID, userID),
+		UpdateExpression:          aws.String("ADD strokes :s"),
+		ExpressionAttributeValues: buildStrokeUpdateExpression(10),
+		ReturnValues:              aws.String("UPDATED_NEW"),
+	})
+	if err != nil {
+		strokeClient.StrokeErrorLogger.Println(err)
+		return "", nil
+	}
+	strokeCount, ok := result.Attributes["strokes"]
+	if ok {
+		return "<@" + userID + "> has " + *strokeCount.N + " strokes.", nil
+	}
+	strokeClient.StrokeErrorLogger.Printf("stroke attribute not found after update guildID=%s userID=%s", guildID, userID)
+	return "", errors.New("stroke attribute not found after update")
+}
+
+// GetStrokesForUser retreives the number of strokes a user has
+func (strokeClient *StrokeClient) GetStrokesForUser(guildID, channelID, userID string) (string, error) {
+	result, err := strokeClient.DynamoClient.GetItem(&dynamodb.GetItemInput{
+		TableName: aws.String(assets.StrikeTableName),
+		Key:       buildStrokeKey(guildID, userID),
+	})
+	if err != nil {
+		strokeClient.StrokeErrorLogger.Println(err)
+		return "", err
+	}
+	strokeCount, ok := result.Item["strokes"]
+	if ok {
+		if *strokeCount.N == "1" {
+			return "<@" + userID + "> has " + *strokeCount.N + " stroke.", nil
```

Comment:
Why are you interpolating if you already know the value?

Reply:
this is copy pasted from strike
```	strikeCount, ok := result.Item["strikes"]
	if ok {
		if *strikeCount.N == "1" {
			return "<@" + userID + "> has " + *strikeCount.N + " strike.", nil```